nextcord/application_command.py

```diff
@@ -510,6 +515,13 @@ def _read_application_commands(self) -> None:
                     value.from_callback(value.callback)
                     self.__cog_application_commands__.append(value)
 
+    def has_error_handler(self) -> bool:
+        """:class:`bool`: Checks whether the cog has an error handler.
+
+        .. versionadded:: 2.4
+        """
+        return not hasattr(self.cog_application_command_error.__func__, "__cog_special_method__")
```

Comment:
I'd say this naming + documentation is slightly misleading as it also provides the assumption the check handles prefix commands also.

I propose renaming it to something like `has_application_command_error_handler` and the documentation accordingly